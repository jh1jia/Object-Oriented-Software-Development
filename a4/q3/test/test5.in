stdin count 5 count l count 3 doublewords doublewords dropfirst 2 dropfirst 1 allcaps count l doublewords count l
HellO world
