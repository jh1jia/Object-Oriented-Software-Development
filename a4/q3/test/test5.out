2O
2O
2O
2O
2O
2O
2O
2O
1D
1D
1D
1D
1D
1D
1D
1D
